### PR TITLE
WRO-697: Create babel-preset-enact to maintain the rule in one place

### PR DESCRIPTION
**Pull request by hong6316**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
There are three repos (cli, storybook-utils, eslint-config-enact) that set the babel config.
It is necessary to change the structure so that this babel config is integrated into one preset and used in three repos.
This makes maintenance and management easier.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Create babel-preset-enact to maintain the rule in one place.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-697

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,10 @@
-# babel-preset-enact-app
-Babel preset used by Enact
+# babel-preset-enact
```
Comment (MikyungKim) [inline README.md:1]: Can we update README with the detailed information? If it cannot be done now, let's create another ticket.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "url": "https://github.com/enactjs/babel-preset-enact.git"
+  },
+  "homepage": "https://enactjs.com",
+  "license": "Apache-2.0",
+  "files": [
+    "index.js"
+  ],
+  "dependencies": {
+    "@babel/plugin-proposal-decorators": "^7.16.4",
+    "@babel/plugin-proposal-export-default-from": "7.18.9",
```
Comment (MikyungKim) [inline package.json:16]: Why some of dependencies were pinned?

**Review by MikyungKim — COMMENTED** [review]
Could we have some lint or prettier and travis setting for this repo?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  "description": "Babel preset used by Enact",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/enactjs/babel-preset-enact.git"
+  },
+  "homepage": "https://enactjs.com",
+  "license": "Apache-2.0",
+  "files": [
+    "index.js"
+  ],
```
Comment (MikyungKim) [inline package.json:13]: Is this necessary?

File: README.md (hunk 1 above)
Comment (hong6316) [reply README.md:1]: OK, I will create another ticket.

File: package.json (hunk 2 above)
Comment (hong6316) [reply package.json]: I think it's copy/paste problem. I will fix them.

**Comment by hong6316** [comment]
> Could we have some lint or prettier and travis setting for this repo?

OK, I will create another ticket.

File: package.json (hunk 3 above)
Comment (hong6316) [reply package.json]: No, I will remove this.

File: README.md (hunk 1 above)
Comment (hong6316) [reply README.md:1]: WRP-7819

File: package.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "babel-preset-enact",
+  "version": "1.0.0",
+  "description": "Babel preset used by Enact",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/enactjs/babel-preset-enact.git"
+  },
+  "homepage": "https://enactjs.com",
```
Comment (MikyungKim) [inline package.json:9]:
```suggestion
  "homepage": "https://github.com/enactjs/babel-preset-enact.git",
```

Comment (hong6316) [reply package.json]: OK

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1,2 @@
+node_modules
```
Comment (MikyungKim) [inline .gitignore:1]:
```suggestion
# Dependency directories
node_modules
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+## unreleased
+
+* Initial internal release
```
Comment (MikyungKim) [inline CHANGELOG.md:3]:
```suggestion
* Initial release
```

File: package.json [hunk 7]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "babel-preset-enact",
+  "version": "1.0.0",
```
Comment (MikyungKim) [inline package.json:3]:
```suggestion
  "version": "0.1.0",
```

File: package.json [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+    "@babel/preset-react": "^7.16.0",
+    "@babel/preset-typescript": "^7.16.0",
+    "@babel/runtime": "^7.16.3",
+    "babel-plugin-dev-expression": "^0.2.3",
+    "babel-plugin-dynamic-import-node": "^2.3.3",
+    "babel-plugin-transform-react-remove-prop-types": "^0.4.24"
+  },
+  "devDependencies": {
+    "eslint": "^8.31.0",
+    "eslint-config-enact": "github:enactjs/eslint-config-enact#feature/WRO-697",
```
Comment (MikyungKim) [inline package.json:42]: Let's use the latest released version so that we can merge this first.

File: .gitignore [hunk 9]
```diff
@@ -0,0 +1,3 @@
+# Dependency directories
+
```
Comment (MikyungKim) [inline .gitignore:2]:
```suggestion
```

File: README.md [hunk 10]
```diff
@@ -1,2 +1,10 @@
...
+
+> A set of babel preset designed by the Enact team.
```
Comment (MikyungKim) [inline README.md:3]:
```suggestion
> This package includes the Babel preset used by Enact.
```

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**